tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -86,6 +86,29 @@ def run(self):
             if is_mon:
                 import_ceph(NS.tendrl_context.integration_id)
         else:
+            # Check if minimum required version of underlying gluster
+            # cluster met. If not fail the import task
+            sds_version = NS.etcd_orm.client.read(
```

Comment:
Please Create the DetectedCluster object and load it, stop using etcd_orm directly for no reason, you do realize that we loose all the base object functions when you do a direct etcd_orm.client.read()

eg: detected_cluster  = NS.tendrl.objects.DetectedCluster().load()
detected_cluster.sds_version....